src-django/api/models.py

```diff
@@ -97,3 +97,70 @@ def save(self, **kwargs):
 
     class Meta:
         app_label = 'api'
+
+
+class Condition(models.Model):
+    CRITERIA_TYPES = (
+        ('EQUALS', 'EQUALS'),
+        ('GREATER', 'GREATER'),
+        ('LESSER', 'LESSER')
+    )
+
+    NODE_TYPES = (
+        ('AND', 'AND'),
+        ('OR', 'OR'),
+        ('NOT', 'NOT'),
+        ('CRITERIA', 'CRITERIA')
+    )
+
+    parent = models.ForeignKey('self', on_delete=models.CASCADE, related_name='children', blank=True, null=True)
+    element = models.ForeignKey(Element, blank=True, null=True)
```

Comment:
Condition != ShowIf. ShowIf is the wrapper around the condition syntax tree. A ConditionNode (if it is a criteria) has a reference to the element it is referring to. I forgot to add the page foreign key to the ShowIf, and will do that.

This also should be nullable, because not all conditions have this. We check to make sure the Criteria ones do though.